Mathlib/CategoryTheory/Sites/CoveringFamilies.lean

```diff
@@ -0,0 +1,116 @@
+/-
+Copyright (c) 2024 Lagrange Mathematics and Computing Research Center. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anthony Bordg
+-/
+import Mathlib.CategoryTheory.Sites.Coverage
+
+/-!
+# Covering Families and their generated topology
+
+Following Laurent Lafforgue, given a category `C`, we introduce the notion
+`CategoryTheory.CoveringFamilies C` of covering families on `C`, i.e. a coverage
+`CategoryTheory.Coverage C` satisfying three additional axioms, and we define
+`CategoryTheory.CoveringFamilies.GeneratedTopology C cf` the topology on `C` generated by a
+collection `cf` of covering families as the topology generated by its underlying coverage.
+
+## Main Results
+
+⋆ `CategoryTheory.CoveringFamilies.covering_of_generatedTopology_iff` : we give an explicit
+  characterization of the covering sieves of the topology generated by covering families.
+
+## References
+
+⋆ Le problème de l'engendrement des topologies de Grothendieck, slides (not publicly available yet)
+  of a talk given by Laurent Lafforgue at Centre Lagrange on 26th April 2024.
+
+## Tags
+
+Grothendieck topology, sieve, generated topology, coverage, covering families
+-/
+
+namespace CategoryTheory
+
+universe u v
+
```

Comment:
```suggestion
universe v u

```

This is the convention in the category theory library. I don't think you will ever need to explicitly specify the universes, so alternatively you may delete them and change the line below to `variable (C : Type*) [Category C]` (that way, Lean introduces arbitrary universe levels automatically).